src/js/controller/qoe.ts

```diff
@@ -7,15 +9,28 @@ import {
     MEDIA_VISUAL_QUALITY
 } from 'events/events';
 import Timer from 'api/timer';
+import type Model from 'controller/model';
+import type { MediaModel } from 'controller/model';
+import type { ProgramController } from 'program/program-controller';
+
+type QoeModel = Model & {
```

Comment:
Should this be `SimpleModel &`?

Reply:
I'm honestly not sure. It looked like the `QoeModel` had all the properties of `model` with a few extra.